dev.goats.xivlauncher.yml

```diff
@@ -1,31 +1,38 @@
 app-id: dev.goats.xivlauncher
+
 runtime: org.freedesktop.Platform
 runtime-version: '22.08'
 sdk: org.freedesktop.Sdk
 sdk-extensions:
   - org.freedesktop.Sdk.Extension.dotnet6
+
 rename-icon: xivlauncher
 copy-icon: true
+
 build-options:
   append-path: "/usr/lib/sdk/dotnet6/bin"
   append-ld-library-path: "/usr/lib/sdk/dotnet6/lib"
   append-pkg-config-path: ":/usr/lib/sdk/dotnet6/lib/pkgconfig"
   env:
     DOTNET_CLI_TELEMETRY_OPTOUT: "true"
 command: xivlauncher
+
 finish-args:
   - --share=ipc
-  - --socket=fallback-x11
-  #- --socket=wayland re-add later when libdecor support is added to builds
+  - --socket=x11 # Wine doesn't support Wayland yet.
+  - --socket=wayland # XIVLauncher does, via ImGui's SDL2 backend.
```

Comment:
I'm aware of some issues with allowing Wayland access, a few people (including myself) have ran into issues with the launcher not correctly displaying a title bar when doing so.

Reply:
This shouldn't be an issue now, I need to update the SDL2 used by xivlauncher anyways. That should fix the titlebar issues too, as well as other Wayland related bugs.